packages/commonwealth/server/workers/cosmosGovNotifications/proposalFetching/v1ProposalFetching.ts

```diff
@@ -72,7 +72,7 @@ export async function fetchUpToLatestCosmosProposalV1(
       });
       proposal = result.proposal;
     } catch (e) {
-      if (!e.message.includes('rpc error: code = NotFound')) {
+      if (!e.message.includes('Request failed with status code 404')) {
```

Comment:
@timolegros A couple notification tests were failing for me. I traced it back to this.
 I think the error this was flagging was anticipated for v1beta1 RPC requests, but not v1 REST requests.

Reply:
Does the new message cover both cases?

Reply:
This new message is just for v1.

Reply:
Ah my bad didn't notice this was just a v1 function.